### DISCUSSION TITLE
Wildcards

### DISCUSSION THREAD

**Discussion by macxrus** [shaitanzx/Fooocus_extend#3] [General]
can I remove the block somehow Wildcards?

**Comment by shaitanzx** [comment]
tell me in more detail what you mean

**Reply by macxrus** [reply]
I just want to disable this section so that I don't see it.

**Reply by shaitanzx** [reply]
Sorry, but this option is not provided. we can discuss this need in the telegram group https://t.me/+xlhhGmrz9SlmYzg6
